core/server/services/email-analytics/jobs/index.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+const config = require('../../../../shared/config');
+const models = require('../../../models');
+const jobsService = require('../../jobs');
+
+let hasScheduled = false;
+
+module.exports = {
+    async scheduleRecurringJobs() {
+        if (
+            !hasScheduled &&
+            config.get('backgroundJobs:emailAnalytics') &&
+            !process.env.NODE_ENV.match(/^testing/)
+        ) {
+            // don't register email analytics job if we have no emails,
+            // processer usage from many sites spinning up threads can be high
+            const emailCount = await models.Email.count();
+
+            if (emailCount > 0) {
```

Comment:
I think we'd need to add a listener for when new emails are added and schedule new jobs once first email is registered.